### PR TITLE
dark mode toggle button added

### PR DESCRIPTION
**Pull request by Dev-Code24**
Hi 
I've added a dark mode toggle button in this project also I've added a video of using the btn
Hopefully, you like it and this PR gets merged

### CODE REVIEW

**Comment by origranot** [comment]
@Dev-Code24 Thank you for the PR!

Do you mind fix the conflict?

**Comment by origranot** [comment]
@Dev-Code24 Hey!

I can't merge this because there is a conflict between the PR and the current master branch.

You can fix it by pulling from master.

**Comment by origranot** [comment]
@Dev-Code24 I can see.

Before merge can you please change couple things?
1. Use single quote instead of double.
2. Make a seperated file for the css (similar to js you did)

**Comment by Dev-Code24** [comment]
@ origranot yeah sure!

**Comment by Dev-Code24** [comment]
@origranot  yeah sure!

File: public/js/index.js [hunk 1]
```diff
@@ -26,29 +38,35 @@ const handleShortenerClick = async () => {
...
-	return result.data;
+  let result;
+  try {
+    result = await axios.post("/api/shortener", {
+      originalUrl,
+    });
+  } catch (err) {
+    return null;
+  }
+  return result.data;
```
Comment (origranot) [inline public/js/index.js:57]: Use single quoute

**Pull request merged**